index.php

```diff
@@ -248,7 +248,7 @@
         }
     }
     $count = count($no_post_users);
-    $countdiff = count($no_post_users) - 10;
+    $countdiff = count($no_post_users) - $i;
```

Comment:
Would this not always be equal to 0? Because $i looks like it's also a count of $no_post_users